backend/mbql/src/metabase/mbql/schema.clj

```diff
@@ -521,6 +521,7 @@
 (defclause <=, field OrderComparible, value-or-field OrderComparible)
 (defclause >=, field OrderComparible, value-or-field OrderComparible)
 
+;; :between is INCLUSIVE just like SQL !!!
 (defclause between field OrderComparible, min OrderComparible, max OrderComparible)
```

Comment:
Is `Comparible` misspelled on purpose? Should be `Comparable`

![image](https://user-images.githubusercontent.com/784417/109184619-cd48f180-7754-11eb-992c-454986775bef.png)